webapi/README.md

```diff
@@ -1,4 +1,4 @@
-# Semantic Kernel Service - CopilotChat
+# Copilot Chat Web API
```

Comment:
```suggestion
# Chat Copilot Web API
```

Reply:
will rename in separate PR, see https://github.com/microsoft/chat-copilot/pull/100#discussion_r1283601447